contentprovider.go

```diff
@@ -364,18 +363,24 @@ func (p *contentProvider) fillContentChunkMatches(ms []*candidateMatch, numConte
 		}
 		firstLineStart := newlines.lineStart(firstLineNumber)
 
+		bestLineMatch := 0
```

Comment:
It is worth adding this to the debugScore?

Reply:
Good idea, I added it to the chunk match debug output. It looks like this:
```
score:5000.00 <- OverlapSymbol:4000.00, kind:Java:classes:1000.00, (line: 3)
```